controllers/prescription.controller.js

```diff
@@ -0,0 +1,121 @@
+const Prescription = require("../models/prescription.model");
+const constants = require("../utils/constants");
+const User = require("../models/user.model");
+
+exports.createPrescription = async()=>{
```

Comment:
Make changes in this file according to the changes made in the model files

Reply:
Doctor should be able to give the prescription for the patient who has taken the appointment